docs/001_develop/03_client-capabilities/014_utility-methods/06_decorators-util.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Decorators Utilities'
+sidebar_label: 'Decorators Utilities'
+id: decorators-utils
+keywords: [utils, utility, data, decorators, env, environment, formatters, number, date, logger, logging, mappers, mixins, observers, resource, server, serializer, json, styles, css, UUID, unique, window, iframe]
+tags:
+  - utils
+  - utility
+  - data
+  - decorators
+  - env
+  - environment
+  - formatters
+  - number
+  - date
+  - logger
+  - logging
+  - mappers
+  - mixins
+  - observers
+  - resource
+  - server
+  - serializer
+  - json
+  - styles
+  - css
+  - UUID
+  - unique
+  - window
+  - iframe
+---
+
```

Comment:
Can you please add a `:::info` box at the top saying decorators only apply to projects using Genesis syntax please

Reply:
Done